workflow/controller/operator.go

```diff
@@ -275,14 +280,24 @@ func (woc *wfOperationCtx) persistUpdates() {
 		return
 	}
 	wfClient := woc.controller.wfclientset.ArgoprojV1alpha1().Workflows(woc.wf.ObjectMeta.Namespace)
+	woc.log.Info("Final size", woc.getSize())
+	if woc.wf.Status.CompressedNodes != "" {
```

Comment:
Maybe we should remove this line  like this

```
        err := woc.checkAndCompress()
	if err != nil {
		woc.log.Warnf("Error compressing workflow: %v", err)
	}
	if woc.wf.Status.CompressedNodes != "" {
		woc.clearNodeStatusMap()
	}
```

Reply:
Thanks updated